app/views/import/index.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Import#index</h1>
```

Comment:
Delete this one, if you use `rails generate controller` delete all the unnecessary files!